packages/react-magma-dom/src/components/NavTabs/NavTab.tsx

```diff
@@ -155,6 +160,15 @@ export const NavTab = React.forwardRef<any, NavTabProps>((props, ref) => {
     ? TabsIconPosition.left
     : TabsIconPosition.top;
 
+  const styledTabRef = React.useRef<HTMLAnchorElement>();
+
+  // Sets focus on first NavTab for accessibility
```

Comment:
Sorry to nitpick, but isFocused can be any tab so we should remove `first` from this comment